### DISCUSSION TITLE
Addressing the speaker from HA

### DISCUSSION THREAD

**Discussion by Mntz** [seaky/nspanel_pro_tools_apk#125] [General]
Is there currently any way to address the speaker from HA?
Maybe the device can be added as a media player?

**Comment by seaky** [comment]
play sound or trigger sound via mqtt is on the backlog

**Reply by Mntz** [reply]
Cool, thanks!

**Comment by CFenner** [comment]
I use the adb integration, with which I get a `media_player` entity. By that I can control the active App. Maybe this can be used?
